src/app/child-dev-project/children/dashboard-widgets/birthday-dashboard/birthday-dashboard.component.ts

```diff
@@ -0,0 +1,57 @@
+import { Component,Input,OnInit,AfterViewInit } from "@angular/core";
+import { EntityMapperService } from "../../../../core/entity/entity-mapper.service";
+import { Child } from "../../model/child";
+import { DynamicComponent } from "../../../../core/view/dynamic-components/dynamic-component.decorator";
+import { OnInitDynamicComponent } from "../../../../core/view/dynamic-components/on-init-dynamic-component.interface";
+import { MatTableDataSource } from "@angular/material/table";
+import { MatSort } from "@angular/material/sort";
+import { MatPaginator } from "@angular/material/paginator";
+import { ViewChild } from "@angular/core";
+import { calculateAge } from "app/utils/utils";
+@DynamicComponent("BirthdayDashboard")
+@Component({
+  selector: "app-birthday-dashboard",
+  templateUrl: "./birthday-dashboard.component.html",
+  styleUrls: ["./birthday-dashboard.component.scss"],
+})
+export class BirthdayDashboardComponent implements OnInitDynamicComponent,AfterViewInit {
+  @Input() dateOfBirth: Date;
+  @Input() age:number;
+
+
+  children: Child[] = [];
+  Child: any ;
+  /**  displayedColumns: string[] = ['entityid', 'dateOfBirth', 'age'];*/
+  childrenDataSource = new  MatTableDataSource<Child>();
+  @ViewChild(MatPaginator) paginator: MatPaginator;
+  @ViewChild(MatSort) sort: MatSort;
+  isLoading: boolean = true;
```

Comment:
You never set `isLoading` to `false`, therefore the statement `...*ngIf="childrenDataSource.data.length > 0 && !isLoading"...` in your template file will also never be true. And the table never be shown.